Framework/DataObjects/src/FakeMD.cpp

```diff
@@ -204,7 +205,7 @@ template <typename MDE, size_t nd> void FakeMD::addFakeEllipsoid(typename MDEven
 
   // prepare random number generator
   std::mt19937 rng(static_cast<unsigned int>(m_randomSeed));
-  std::normal_distribution<double> d(0.0, 1.0); // mean = 0, std = 1
+  Kernel::normal_distribution<double> d(0.0, 1.0); // mean = 0, std = 1
```

Comment:
Good spot that `normal_distribution` is also called in the ellipsoid peak simulation - this is probably the root of inconsistent behaviour between OS in ellipsoid tests for `IntegratePeaksMD` that I've struggled with in the past.